daemon/server/router/container/inspect.go

```diff
@@ -36,23 +35,27 @@ func (c *containerRouter) getContainersByName(ctx context.Context, w http.Respon
 		ctr.ImageManifestDescriptor = nil
 	}
 
-	var bridgeNw network.EndpointSettings
-	if v := ctr.NetworkSettings.Networks["bridge"]; v != nil {
-		bridgeNw = *v
-	}
-
 	var wrapOpts []compat.Option
 	if versions.LessThan(version, "1.52") {
-		wrapOpts = append(wrapOpts, compat.WithExtraFields(map[string]any{
-			"EndpointID":          bridgeNw.EndpointID,
-			"Gateway":             bridgeNw.Gateway,
-			"GlobalIPv6Address":   bridgeNw.GlobalIPv6Address,
-			"GlobalIPv6PrefixLen": bridgeNw.GlobalIPv6PrefixLen,
-			"IPAddress":           bridgeNw.IPAddress,
-			"IPPrefixLen":         bridgeNw.IPPrefixLen,
-			"IPv6Gateway":         bridgeNw.IPv6Gateway,
-			"MacAddress":          bridgeNw.MacAddress,
-		}))
+		if bridgeNw := ctr.NetworkSettings.Networks["bridge"]; bridgeNw != nil {
+			// Old API versions showed the bridge's configuration as top-level
+			// fields in "NetworkConfig".
+			//
+			// This was deprecated in API v1.44, but kept in place until
+			// API v1.52, which removes this entirely.
+			wrapOpts = append(wrapOpts, compat.WithExtraFields(map[string]any{
+				"NetworkSettings": map[string]any{
+					"EndpointID":          bridgeNw.EndpointID,
+					"Gateway":             bridgeNw.Gateway,
+					"GlobalIPv6Address":   bridgeNw.GlobalIPv6Address,
+					"GlobalIPv6PrefixLen": bridgeNw.GlobalIPv6PrefixLen,
+					"IPAddress":           bridgeNw.IPAddress,
+					"IPPrefixLen":         bridgeNw.IPPrefixLen,
+					"IPv6Gateway":         bridgeNw.IPv6Gateway,
+					"MacAddress":          bridgeNw.MacAddress,
```

Comment:
So, we don't add an extra `MacAddress` field to the `Config` struct because  we forgot to add this migration in https://github.com/moby/moby/pull/47233?